### DISCUSSION TITLE
Migrator - the value is outside the range that can be represented as an int

### DISCUSSION THREAD

**Discussion by MvdO79** [wheels-dev/wheels#1440] [Q&A] [answered]
Any idea's on below error message? Where to look for and where?

![2024-05-20 13_07_18-Migrator _ CFWheels en nog 9 andere pagina's - Persoonlijk - Microsoft​ Edge](https://github.com/cfwheels/cfwheels/assets/14022664/72fc7bdf-e53a-4578-9f96-4144b5f48874)

**Comment by neokoenig** [comment, marked as answer]
It's being called from the obfuscateParam function. Have you got that enabled? Might be worth trying turning it off if so

**Reply by MvdO79** [reply]
Thanks, that solved the error 👍

**Comment by MvdO79** [comment]
Do I need to report this as an error?

**Reply by neokoenig** [reply]
I'd say so, That function really has no place in the internal wheels migration CFC.
But it's a limitation of how those functions are imported at present.

**Reply by MvdO79** [reply]
Ok thanks again, will report it for future enhancement.
